tests/robustness/client/kvhash.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2025 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"go.uber.org/zap"
+
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/tests/v3/framework/e2e"
+	"go.etcd.io/etcd/tests/v3/robustness/identity"
+)
+
+func getHashKV(ctx context.Context, t *testing.T, endpoints []string, rev int64) (int64, *clientv3.HashKVResponse) {
+	c, err := clientv3.New(clientv3.Config{
+		Endpoints:            endpoints,
```

Comment:
Again endpoints in client don't need to match endpoints in hashKV call. You don't need to create a client per call!

Reply:
Good point, refactored.